docs/ansible.utils.validate_filter.rst

```diff
@@ -49,9 +49,9 @@ Parameters
                     <td>
                     </td>
                 <td>
-                        <div>The criteria used for validation of value that represents <code>data</code> options.</div>
-                        <div>This option represents the first argument passed in the filter plugin For example <em>config_data|ansible.utils.validate(config_criteria</em>), in this case the value of <em>config_criteria</em> represents this option.</div>
-                        <div>For the type of <code>criteria</code> that represents this value refer documentation of individual validator plugins.</div>
+                        <div>The criteria used for validation of value that represents <code>data</code>. options.</div>
```

Comment:
Same question here - it looks like a stray `.` or a search/replace error, maybe?